Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG VERSION=nonroot
 
 # Build the manager binary
-FROM golang:1.15 as builder
+FROM golang:1.17 as builder
```

Comment:
why are you bumping this?

Reply:
No technical reason, just to follow the latest Go version.

Reply:
I would rather postpone this till we get more time to properly test it on that version, JIC